<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-background-color: rgb(250, 247, 187);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_pro">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;b&gt;&lt;i&gt;Tetris by Tom &amp; Nghia&lt;/i&gt;&lt;/b&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_nv">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Niveau : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_score">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Score :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_duree">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Durée : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ligne">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Lignes :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_apm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>&lt;b&gt;Statistiques&lt;/b&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_nv_val">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;b&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_score_val">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ligne_val">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;b&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_duree_val">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;b&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Démarrer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Pause/Reprendre</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionManuel_d_utilisaleur"/>
    <addaction name="actionManuel_de_d_veloppeurs"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManuel_d_utilisaleur">
   <property name="text">
    <string>Manuel d'utilisaleur</string>
   </property>
  </action>
  <action name="actionManuel_de_d_veloppeurs">
   <property name="text">
    <string>Manuel de développeurs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
